### ISSUE TITLE
UI shows release notes twice

### ISSUE THREAD

**Issue by Botan626** [JustArchiNET/ASF-ui#722]
![image](https://user-images.githubusercontent.com/28785698/63832201-f8dcf700-c978-11e9-9822-ba32a71b6d93.png)

**Comment by Botan626** [comment]
![image](https://user-images.githubusercontent.com/28785698/64156765-13621500-ce3e-11e9-8e43-ed57da1e35ad.png)

**Comment by Botan626** [comment]
![image](https://user-images.githubusercontent.com/28785698/65580535-86186900-df82-11e9-9349-3947678f03de.png)

**Comment by Abrynos** [comment]
@Botan626  wanna do it again? i'm not sure everyone has seen yet
![doitagain](https://user-images.githubusercontent.com/6608231/65633163-c35d1500-dfdb-11e9-999d-e2922dc16f37.png)

**Comment by guixxx** [comment]
Hm, I couldn't reproduce this here... Do you know any consistent way of doing it?

**Comment by Botan626** [comment]
I'm not sure, but I think it's gone.

**Comment by Botan626** [comment]
Actually this came back.
![image](https://user-images.githubusercontent.com/28785698/67642482-3fb97f80-f91d-11e9-8808-d4389452f283.png)

What info I can provide which will be helpful?

**Comment by Botan626** [comment]
![image](https://user-images.githubusercontent.com/28785698/67642518-9de66280-f91d-11e9-9b19-67ed0794b2bd.png)

**Comment by Botan626** [comment]
[{"event_id":"dfc7a936065c4c5fbf36057246dccb9b","level":"info","message":"Snapshot","platform":"javascript","sdk":{"name":"sentry.javascript.browser","packages":[{"name":"npm:@sentry/browser","version":"4.6.5"}],"version":"4.6.5"},"environment":"production","release":"b306d3381fa64afe99fe6c8fa732c409ad7b4007","tags":{"asf.version":"4.1.0.3","asf.buildVariant":"win-x64"},"fingerprint":["Snapshot"],"breadcrumbs":[{"timestamp":1572213309.105,"category":"xhr","data":{"method":"GET","url":"/api/bot/asf","status_code":200},"type":"http"},{"timestamp":1572213309.585,"category":"ui.click","message":"div.form-item.form-item--config > div.form-item__value > div.form-item__buttons"},{"timestamp":1572213310.738,"category":"ui.click","message":"span.dropdown__label"},{"timestamp":1572213311.605,"category":"xhr","data":{"method":"GET","url":"/api/bot/asf","status_code":200},"type":"http"},{"timestamp":1572213314.104,"category":"xhr","data":{"method":"GET","url":"/api/bot/asf","status_code":200},"type":"http"},{"timestamp":1572213316.168,"category":"ui.click","message":"ul.dropdown__items > li.dropdown__item"}],"request":{"url":"http://127.0.0.1:1242/page/ui-configuration","headers":{"User-Agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.70 Safari/537.36"}},"extra":{"store":{"asf":{"memoryUsage":15859,"startTime":"2019-10-26T12:17:16.280Z","buildVariant":"win-x64","version":"4.1.0.3","uptime":"1 d 9 h 38 m","updateChannel":2},"auth":{"status":"AUTHENTICATED"},"bots":{"bots":{"01":{"name":"01","nickname":"Botan626","steamid":"76561197960895331","avatarHash":"d4d42c066c8223beb657e3841f08d958e7d4995b","bgrCount":0,"active":true,"config":{"AcceptGifts":true,"AutoSteamSaleEvent":true,"Enabled":true,"FarmingOrders":[3],"GamesPlayedWhileIdle":[515040],"HoursUntilCardDrops":4,"LootableTypes":[],"MatchableTypes":[],"Paused":true,"RedeemingPreferences":5,"TradingPreferences":1,"TransferableTypes":[],"HandleOfflineMessages":false},"isConnected":true,"paused":true,"gamesToFarm":[],"timeRemaining":"00:00:00","currentGamesFarming":[]},"02":{"name":"02","nickname":"Botan727","steamid":"76561197962691600","avatarHash":"8fc03b52c03719b5a83cd33798f0bea4cb27e652","bgrCount":0,"active":true,"config":{"AcceptGifts":true,"AutoSteamSaleEvent":true,"BotBehaviour":7,"Enabled":true,"FarmingOrders":[3],"LootableTypes":[0,1,2,3,4,5,6,7,8,9],"MatchableTypes":[],"SendTradePeriod":1,"SteamUserPermissions":{"76561197984489752":1,"76561197960895331":3,"76561197964164954":1,"76561197962364358":1,"76561198046025442":1,"76561198387401820":1}},"isConnected":true,"paused":false,"gamesToFarm":[],"timeRemaining":"00:00:00","currentGamesFarming":[]},"03":{"name":"03","nickname":"Botan828","steamid":"76561197962364358","avatarHash":"27025c5c9e433ab1990ee95f6893e0a862da8a7d","bgrCount":0,"active":true,"config":{"AcceptGifts":true,"AutoSteamSaleEvent":true,"BotBehaviour":7,"Enabled":true,"FarmingOrders":[3],"LootableTypes":[0,1,2,3,4,5,6,7,8,9],"MatchableTypes":[],"SendTradePeriod":1,"SteamUserPermissions":{"76561197984489752":1,"76561197960895331":3,"76561197964164954":1,"76561197962691600":1,"76561198046025442":1,"76561198387401820":1},"TradingPreferences":1},"isConnected":true,"paused":false,"gamesToFarm":[],"timeRemaining":"00:00:00","currentGamesFarming":[]},"04":{"name":"04","nickname":"Botan929","steamid":"76561197964164954","avatarHash":"8fc03b52c03719b5a83cd33798f0bea4cb27e652","bgrCount":0,"active":true,"config":{"AcceptGifts":true,"AutoSteamSaleEvent":true,"BotBehaviour":7,"Enabled":true,"FarmingOrders":[3],"LootableTypes":[0,1,2,3,4,5,6,7,8,9],"MatchableTypes":[],"SendTradePeriod":1,"SteamUserPermissions":{"76561197984489752":1,"76561197960895331":3,"76561197962364358":1,"76561197962691600":1,"76561198046025442":1,"76561198387401820":1},"TradingPreferences":1},"isConnected":true,"paused":false,"gamesToFarm":[],"timeRemaining":"00:00:00","currentGamesFarming":[]},"05":{"name":"05","nickname":"Botan","steamid":"76561197984489752","avatarHash":"76b1de8e0ec98be70a114e027f72f4ec2e5c1512","bgrCount":0,"active":true,"config":{"AcceptGifts":true,"AutoSteamSaleEvent":true,"BotBehaviour":7,"Enabled":true,"FarmingOrders":[3],"GamesPlayedWhileIdle":[515040],"HoursUntilCardDrops":1,"LootableTypes":[0,1,2,3,4,5,6,7,8,9],"MatchableTypes":[],"SendTradePeriod":1,"SteamUserPermissions":{"76561197960895331":3,"76561197964164954":1,"76561197962364358":1,"76561197962691600":1,"76561198046025442":1,"76561198387401820":1},"TradingPreferences":1},"isConnected":true,"paused":false,"gamesToFarm":[],"timeRemaining":"00:00:00","currentGamesFarming":[]},"06":{"name":"06","nickname":"Botan10210","steamid":"76561198387401820","avatarHash":"8fc03b52c03719b5a83cd33798f0bea4cb27e652","bgrCount":0,"active":true,"config":{"AcceptGifts":true,"AutoSteamSaleEvent":true,"BotBehaviour":7,"Enabled":true,"FarmingOrders":[3],"GamesPlayedWhileIdle":[515040],"LootableTypes":[0,1,2,3,4,5,6,7,8,9],"MatchableTypes":[],"SendTradePeriod":1,"SteamUserPermissions":{"76561197984489752":1,"76561197960895331":3,"76561197964164954":1,"76561197962364358":1,"76561197962691600":1,"76561198046025442":1},"TradingPreferences":1},"isConnected":true,"paused":false,"gamesToFarm":[],"timeRemaining":"00:00:00","currentGamesFarming":[]},"07":{"name":"07","nickname":null,"steamid":"0","avatarHash":"0b46945851b3d26da93a6ddba3ac961206cc191d","bgrCount":0,"active":false,"config":{"AcceptGifts":true,"AutoSteamSaleEvent":true,"BotBehaviour":7,"FarmingOrders":[3],"LootableTypes":[0,1,2,3,4,5,6,7,8,9],"MatchableTypes":[],"SendTradePeriod":1,"SteamUserPermissions":{"76561197960895331":3},"TradingPreferences":1,"Name":"07"},"isConnected":false,"paused":false,"gamesToFarm":[],"timeRemaining":"00:00:00","currentGamesFarming":[]}}},"layout":{"smallNavigation":false,"sideMenu":false,"theme":"red","availableThemes":["blue","red","teal","purple","green","orange"],"boxed":false,"darkMode":true},"settings":{"defaultView":"commands","notificationPosition":"rightBottom","notifyRelease":true,"nicknames":false,"gameName":false,"favButtons":9,"displayCategories":false,"sentryInstalled":true,"sentryReporting":false,"selectedBots":[],"timestamps":true},"i18n":{"locale":"en-US","fallbackLocale":"en-US"}}}}]

**Comment by Botan626** [comment]
![image](https://user-images.githubusercontent.com/28785698/68329966-78a6e080-00e3-11ea-817e-d398dc85ce55.png)

**Comment by Botan626** [comment]
![image](https://user-images.githubusercontent.com/28785698/68380229-7b93e680-0160-11ea-9b7b-4cf7f56b11ea.png)

As you can see, the uptime for ASF is almost 14 hours, I decided to give it some time in hope the issue could resolve itself, but it didn't.

**Comment by Botan626** [comment]
I downloaded and tested this ASF-ui build, did perform ctrl+F5 on the ui page:
https://ci.appveyor.com/project/JustArchi/asf-ui/builds/28757974/artifacts

And still saw duplicated release note.

But then I manually deleted cookies for here:
![image](https://user-images.githubusercontent.com/28785698/68575479-38e24f00-047d-11ea-9522-4a69ed9779fc.png)

And the issue was gone.

FYI, my default page in ui was commands page.
